plugins/ckeditor5-woltlab-mention/src/woltlabmention.ts

```diff
@@ -28,7 +27,7 @@ export class WoltlabMention extends Plugin {
   }
 
   static get requires() {
-    return [Mention, WoltlabMetacode] as const;
+    return [WoltlabMetacode] as const;
```

Comment:
This change looks wrong, why did you drop the dependency when we are obviously depending on it?